examples/gallery/3d_plots/scatter3d.py

```diff
@@ -36,9 +41,9 @@
 
 # Define a colormap to be used for three categories, define the range of the
 # new discrete CPT using series=(lowest_value, highest_value, interval),
-# use color_model="+c" to write the discrete color palette "cubhelix" in
-# categorical format
-pygmt.makecpt(cmap="cubhelix", color_model="+c", series=(0, 3, 1))
+# use color_model="+c" + species to write the discrete color palette "cubhelix" in
+# categorical format and add the species names extracted above as annotations
+pygmt.makecpt(cmap="cubhelix", color_model="+c" + species, series=(0, 2, 1))
 
 fig.plot3d(
```

Comment:
Could you also add a title to this 3d plot please? We wanted to add it before (see https://github.com/GenericMappingTools/pygmt/pull/471#discussion_r516387507) but it didn't look good without the back walls (that only comes with GMT 6.2).

You can do so by changing `"WsNeZ3"` to `'WsNeZ3+t"Iris flower data set"'`. Feel free to change the title to something better if you want :smile:.

Reply:
Done, looks much better using GMT 6.2. 👍.